.github/workflows/ci-develop.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
     - id: install
       name: Install dependencies
       run: |-
+        git config --global url."https://".insteadOf ssh://
```

Comment:
Added this to change `npm` to use  `https` instead of `git` which requires an SSH key, which is not set up in the CI machine.

Reply:
Reference: https://www.programmersought.com/article/94636987109/